target/linux/rockchip/patches-6.6/401-arm64-dts-rockchip-rk3566-Add-Nanopi-R3S.patch

```diff
@@ -0,0 +1,658 @@
+--- /dev/null
++++ b/arch/arm64/boot/dts/rockchip/rk3566-nanopi-r3s.dts
```

Comment:
It seems to be a mix of upstream nanopi-r5s dts and vendor nanopi-r3s dts.

However the regulator parts is simpler on r3s, there's no `vcc3v3_pcie` and `vcc5v0_usb_*`. For other parts, like `usb2phy1*` is disabled by default so you don't need to disable them again.

I have sent nanopi r3s support to upstream kernel, could you use that version instead?
https://lore.kernel.org/linux-rockchip/20241020173946.225960-2-cnsztl@gmail.com/

Addtionally, please create a separated patch to add OpenWrt's LED aliases, fyi:
https://github.com/openwrt/openwrt/blob/913368a223c3818d2b5b60b06820532038be3467/target/linux/rockchip/patches-6.6/118-arm64-dts-rockchip-Update-LED-properties-for-Radxa-Ro.patch#L19-L22

Reply:
Done - thanks for the feedback. I have refactored the patches using your upstream-submitted DTS, added a separate patch for the LED aliases, and realphabetized 02_network. I've also tested a new build with the aforementioned changes and can confirm that things work as expected.